native/packages/react-native-bpk-component-select/src/BpkSelect.js

```diff
@@ -72,18 +81,30 @@ const TouchablePlatformComponent = Platform.select({
 });
 
 type Props = {
-  onPress: () => mixed,
   disabled: boolean,
   label: ?(string | Element),
+  onPress: () => mixed,
```

Comment:
Tiny thing but can `onPress` go above `label`? For Flow types we do non-nullable ones first, nullable ones afterwards.